feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.ts

```diff
@@ -102,7 +102,7 @@ export class CheckoutDeliveryModesService
                     DeliveryModeSetEvent
                   );
                   /**
-                   * TODO: We have to keep this here, since the cart feature is still ngrx-based.
+                   * TODO:#checkout We have to keep this here, since the cart feature is still ngrx-based.
```

Comment:
same case where I mentioned #deprecation-checkout and file being similar to

https://github.com/SAP/spartacus/pull/14479/files#diff-a80b49078c22389d372ba910f61279fd49ae5a3b53651c26807b4ab900e4fd5eR41

Reply:
We will replace this with a facade call.